ignite/pkg/common/common.go

```diff
@@ -0,0 +1,255 @@
+package common
```

Comment:
The package name should be more specific, maybe it could be named as `codeanalisys`, `appanalisys` or `codeinspect`. Its purpose could be to provide a **simpler** or unified API to inspect/analize blockchain apps using the other specific analysis libraries within `pkg`.

It would be ideal to have it very well documented if the point is to make it easier for plugin writers to understand how to inspect the app code.